docs/config.rst

```diff
@@ -41,3 +41,19 @@ An environment variable like ``REACT_APP_FOO`` controls the setting ``FOO``.
     :default: ``https://${OIDC_DOMAIN}/userinfo``
 
    The audience for the access token that is generated by Auth0.
+
+
+.. envvar:: REACT_APP_REFRESH_AUTH_SESSION_SECONDS
+
+    :default: 300
+
+    How often to issue a silent authentication refresh.
+
+.. envvar:: REACT_APP_REFRESH_PREEMPTIVE_SECONDS
+
+    :default: 900
+
+    How soon *before* the session expires do we perform a silent authentication to keep it fresh.
+    Setting this liberally means we can make sure the access token is up-to-date *before* the
+    user attempts to submit something to a backend and then avoid the access token being
+    out of date.
```

Comment:
I don't understand how both settings interact with each other. Are you saying we need a new silent auth refresh every 300 seconds, but also 900 seconds before the session expires?
It seems weird to have the first one being shorter than the second one, which feels like the second one is useless?

Reply:
Or maybe it's a naming issue? Is it "how often should we check if it's time to refresh" for the first var, and "how long before the expiry is it time to refresh"?

So maybe they could be named something like `REACT_APP_CHECK_SESSION_EVERY_SECONDS` and `REACT_APP_REFRESH_BEFORE_SECONDS`?

I still feel those two could be one and unique, I'm not sure I see the benefit in having both.

Reply:
Yeah, the intention is that there are two numbers. One is how often long the sleep is in the infinite loop, the other is what is too close to the end of the expires. Inside the code I use the word "preemptive" which is basically a way to make sure the accesstoken is refreshed *before* it expires. To your point, the word "before" is probably better. 

I'll try to review the language and update.

Reply:
@magopian Have a look again. I only changed the documentation text. Is it clearer now?